frontend/src/styles/components/_ApartmentDetails.sass

```diff
@@ -42,7 +42,7 @@
         content: "|"
 
     &--number
-      padding: $spacing-xs $spacing-s
+      padding: $spacing-xs $spacing-s                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               3
```

Comment:
What's happening here? 😄

Reply:
No idea. :D I've stumbled on "changes" like this sometimes before with git.